.buildkite/pipeline.yml

```diff
@@ -21,63 +21,49 @@ env:
   USE_NODE_12: true
 
 steps:
-  - label: ':npm: Install Dependencies'
-    command: 'buildkite-scripts/mars/wrap/npm-ci.sh'
-
-  - wait
-
-  - label: ':npm: :lock: NPM Audit'
-    command: '.buildkite/scripts/npm-audit.sh'
-
-  - wait
-
-  - label: ':npm: Tests'
-    command: 'buildkite-scripts/scripts/npm-test.sh'
-    artifact_paths:
-      - 'coverage/lcov.info'
-
-  - wait
-
-  - label: ':house_with_garden: Create block for deploy environment selection'
-    branches: 'feature/* hotfix/*'
-    command: 'buildkite-scripts/mars/pipeline-steps/get-and-select-develop-environments.sh | buildkite-agent pipeline upload'
-
-  - wait
-
-  - label: ':house_with_garden: Set Environment'
-    command: 'buildkite-scripts/mars/scripts/set-deploy-environment.sh'
-    branches: 'master release/* feature/* hotfix/*'
+  - label: ':gear: build & unit test'
+    commands:
+      - buildkite-agent pipeline upload .buildkite/pre.yml
 
   - wait
 
   - label: ':gear: :rocket: develop'
-    commands:
-      - buildkite-agent pipeline upload .buildkite/ci.yml
-    env:
-      BLOCK_DEPLOY: 'build.branch =~ /^feature\//'
-      VIRTRU_ENV: develop01
-      S3_PATH: protect-develop
+    trigger: protect-and-track-deploy
+    build:
+      message: '[develop] ${BUILDKITE_MESSAGE}'
+      commit: '${BUILDKITE_COMMIT}'
+      branch: '${BUILDKITE_BRANCH}'
+      env:
+        BLOCK_DEPLOY: 'build.branch =~ /^feature\//'
+        VIRTRU_ENV: develop01
```

Comment:
Should we allow devs to select the environment to deploy to?

Reply:
yeah probably but I'll do that later so not to block the rollout. Filed https://virtru.atlassian.net/browse/PLAT-329